src/modules/Staff/Api/Guest.php

```diff
@@ -97,4 +97,39 @@ public function login($data)
 
         return $this->getService()->login($data['email'], $data['password'], $this->getIp());
     }
+
+    public function passwordreset($data){
+        $required = [
+            'email' => 'Email required',
+        ];
+        $validator = $this->di['validator'];
+        $validator->checkRequiredParamsForArray($required, $data);
+        $data['email'] = $this->di['tools']->validateAndSanitizeEmail($data['email']);
+        $c = $this->di['db']->findOne('Admin', 'email = ?', [$data['email']]);
+
+        if (!$c instanceof \Model_Admin) {
+            throw new \Box_Exception('Email not found in our database');
+        }
+        $hash = hash('sha256', time().random_bytes(13));
+
+        $c->pass = $hash;
+
+        $reset = $this->di['db']->dispense('AdminPasswordReset');
+        $reset->admin_id = $c->id;
+        $reset->ip = $this->ip;
+        $reset->hash = $hash;
+        $reset->created_at = date('Y-m-d H:i:s');
+        $reset->updated_at = date('Y-m-d H:i:s');
+        $this->di['db']->store($reset);
+
+        // send email
+        $email = [];
+        $email['to_client'] = $reset->admin_id;
```

Comment:
This needs to change so it is send to the admin user and not the client

Client ID 1 is might be different then the admin "User" ID

Reply:
Should be to_staff instead of "to_client"